Ports/vim/package.sh

```diff
@@ -15,7 +15,9 @@ export vim_cv_tgetent=zero
 export vim_cv_terminfo=yes
 export vim_cv_toupper_broken=no
 export vim_cv_tty_group=world
-
+export vim_cv_uname_output=SerenityOS
+export vim_cv_uname_r_output=1.0-dev
+export vim_cv_uname_m_output=i686
```

Comment:
We should use `SERENITY_ARCH` here instead of hardcoding `i686`.

Reply:
Thank you, done!